src/main/java/InfrastructureManager/Rest/RestInput.java

```diff
@@ -0,0 +1,17 @@
+package InfrastructureManager.Rest;
+
+import InfrastructureManager.MasterInput;
+import spark.*;
+
+public class RestInput implements MasterInput {
+    private static String command = "";
+    public static Route readCommand = (Request request, Response response) ->{
+        command = request.params(":command");
+        return command;
+    };
+
+    @Override
+    public String read() {
+        return command;
```

Comment:
good point, actually I'm not really sure how to implement the read method for REST since most inputs would be answered through the Route objects

I just returned command to implement the read method, would it be better to leave it blank or return void instead? @alyhasansakr @juan-castrillon

Reply:
assuming there's no command, is it a good practice to return nothing in Java?

Reply:
throw an exception instead.